.github/issue_reporting_guide.md

```diff
@@ -0,0 +1,267 @@
+# Issue reporting guide
+
+* [Security issues](#security-issues)
+* [Search for an existing issue](#search-for-an-existing-issue)
+* [Reproducible test case](#reproducible-test-case)
+* [Describe your environment](#describe-your-environment)
+* [Reporting deadlock](#reporting-deadlock)
+* [Reporting panics/error returns](#reporting-panics-error-returns)
+* [Gathering more debugging data](#gathering-more-debugging-data)
+* [Regressions](#regressions)
+* [Debugging issues that only reproduce in the live environment](#debugging-issues-that-only-reproduce-in-the-live-environment)
+* [Follow the progress of your issue](#follow-the-progress-of-your-issue)
+* [Try fixing your issue yourself](#try-fixing-your-issue-yourself)
+* [Additional reading](#additional-reading)
+
+## Security issues
+
+Do NOT report security issues publicly. See [https://github.com/moby/buildkit/security/policy](https://github.com/moby/buildkit/security/policy) 
+
+## Search for an existing issue
+
+Before creating a new issue, search the repository if a similar issue has already been reported before. You can also try Google, as Github search is not that great.
+
+If you find an existing issue, there is still a chance that you can provide additional information there. It may be missing a lot of information you can provide that could help contributors fix the problem quicker. This guide should give you enough information to evaluate if the existing report could be improved.
+
+You can use the reactions feature of Github to signal that you have also hit the same problem. Leaving “me too” comments on the issue without any new useful information isn’t really helpful. If you notice that the issue is not properly labeled or should be in the milestone (see “Follow the progress of your issue”) then you can leave a comment, notifying the maintainers. If the issue you find is already closed, it is usually better to create a new one instead of leaving a new comment on closed issue.
```

Comment:
```suggestion
You can use the reactions feature of Github to signal that you have also hit the same problem. Leaving “me too” comments on the issue without any new useful information isn’t really helpful. If you notice that the issue is not properly labeled or should be in the milestone (see [Follow the progress of your issue](#follow-the-progress-of-your-issue)) then you can leave a comment, notifying the maintainers. If the issue you find is already closed, it is usually better to create a new one instead of leaving a new comment on closed issue.
```